literature/hamad_2003_electrosorption_211/hamad_2003_electrosorption_211_p2_1a_solid.yaml

```diff
@@ -0,0 +1,102 @@
+    comment: combined with Monte-Carlo simulation # Add whatever you think is important to understand the data
+
+electrochemical system:
+    version: 1
+    instrumentation:
+        - type: 
+          supplier: 
+        - type:
+          model:
+          supplier:
+        - type:
+          model:
+          supplier:
+    electrolyte:
+        type: aq # aqueous = aq, non-aqueous=naq
+        components: # create as many components as necessary. A simple electrolyte consists of two components, i.e., H2SO4 and water
+            - name: KClO4 # can be trivia name, sum formula, etc
+              concentration:
+                  value: 0.03
+                  unit: M #[M, mM, µM, g kg-1, ...]
+              type: salt #
+              source:
+                  supplier:
+                  LOT: 
+            - name: water
+              type: solvent
+              proportion:
+                  value:
+                  unit:
+              source:
+                  refinement:
+                  quality:
+                  total organic carbon: 
+                      value:
+                      unit:
+            - name: 
+              type:
+              proportion:
+                  value:
+                  unit:
+            - name: KCl
+              type: salt
+              source:
+                  supplier:
+                  LOT: 
+              concentration:
+                  value: 0.02
+                  unit: M #[M, mM, µM, g kg-1, ...] 
+        ph: 
+            value:
+            uncertainty:
+        temperature: # provide the temperature as it is given in the article. Assume room temperature if nothing is provided
+            value: 293 
+            unit: K # [K, C, F]
+    electrodes:
+        configuration: 3 # number of electrodes
+        reference electrode:
+            type: SCE # the reference electrode given in the experimental section. Might differ from that in the plot, MSE-sat, MSE-3M, MSE-1M, RHE, AgCl2
+            source:
+                supplier: SI Analytics
+        counter electrode:
+            material: Pt
+            crystallographic orientation:
+            shape: wire
+        working electrode:
+            material: Ag
+            crystallographic orientation: '100' # hkl, 100, 110, poly
+            source:
+                supplier: 
+                LOT: 
+            shape: # optional
+                type: # bead/sphere
+                height:
+                    value:
+                    unit:
+                diameter:
+                    value:
+                    unit:
+            geometric electrolyte contact area: 
+                value: 
+                unit: 
+            preparation procedure: electrode etched with cyanide solution, rinsed with Milli-Q water, annealed in a H2 flame for 30s. # Short description of sample preparation. This can be very useful. :)

```

Comment:
```suggestion
            preparation procedure: Electrode etched with a cyanide solution, rinsed with Milli-Q water, annealed in a H2 flame for 30 s. # Short description of sample preparation. This can be very useful. :)

```